### PR TITLE
OF-1566: Fix for archive search with password protected chatrooms

### PR DESCRIPTION
**Pull request by NielsKr**
Fix for preventing users to access message history (archive search) of a chatroom when they are no occupant in the chatroom at the moment of search.
The fix results in a Forbidden response being send when the requestor of the archive search is not an occupant in the chatroom.

See [Openfire 4.2.3 monitoring service 1.6.0 does not check user has the right to enter the archive] (https://discourse.igniterealtime.org/t/openfire-4-2-3-monitoring-service-1-6-0-does-not-check-user-has-the-right-to-enter-the-archive/82037)

### CODE REVIEW

File: src/plugins/monitoring/plugin.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <name>Monitoring Service</name>
     <description>Monitors conversations and statistics of the server.</description>
     <author>IgniteRealtime // Jive Software</author>
-    <version>1.6.0</version>
+    <version>1.6.1</version>
     <date>03/05/2018</date>
```
Comment (guusdk) [inline src/plugins/monitoring/plugin.xml]: Probably should change the date here.

**Comment by guusdk** [comment]
Thanks for this! Could you update the release date please? Otherwise, the changes look good to me.

File: src/plugins/monitoring/plugin.xml (hunk 1 above)
Comment (NielsKr) [reply src/plugins/monitoring/plugin.xml]: Updated the release date and included issue number in changelog.

**Comment by guusdk** [comment]
@akrherz would you mind pushing a new plugin version to the website?

**Comment by akrherz** [comment]
@guusdk thanks for your bamboo hint, this is done.

**Pull request merged**